vm/vm.go

```diff
@@ -773,6 +773,8 @@ func (vm *VM) call(caller, callee *Account, code, input []byte, value int64, gas
 				if err != nil {
 					exception = err.Error()
 				}
+				// NOTE: these fire call events and not particular events for eg name reg or permissions
+				// TODO: change this?
```

Comment:
presumably any js client. i suspect theyll mostly transact through the CallTx, especially since contracts will be set up for more fine grained permissioning of name/permissions access. We can just put the burden on them to treat a call event to the nametx address (doesnt exist yet) or the permissions addresses as what they are - I wouldn't be opposed